### ISSUE TITLE
The cursor gets stuck in the collapsed YAML zone

### ISSUE THREAD

**Issue by z2k-gwp** [hua03/obsidian-toggle-meta-yaml-plugin#6]
Thanks for providing this functionality - its been wished for many many times!

I noticed one strange thing though - if you use the cursor keys to move around, then when the cursor gets to where the collapsed header is you can't use the down arrow to escape out of it. You must go to the mouse and click out of the collapsed area. 

Given that it is at the top, I constant find myself going to the top of the document with a keyboard command and then getting stuck there.

**Comment by z2k-gwp** [comment]
By the way, this css solution manages to keep the cursor from not getting stuck. But then, it doesnt exactly hide the yaml either.

```css
.is-live-preview .cm-line .cm-hmd-frontmatter {
  color: var(--text-muted);
}
.is-live-preview .cm-line:not(.cm-active) > .cm-hmd-frontmatter:not(.cm-meta) {
  font-size: 0.8em;
  color: inherit;
  font-weight: 600;
  border-radius: 0.8em;
  padding: 0.2em 0.8em;
  background-color: var(--background-secondary-alt);
}
.is-live-preview .cm-line > .cm-hmd-frontmatter:not(.cm-meta) .cm-selection,
.is-live-preview .cm-line > .cm-hmd-frontmatter:not(.cm-meta).cm-selection {
  color: var(--text-muted);
  background-color: var(--text-selection) !important;
  font-weight: initial;
}

.is-live-preview .cm-line:not(.cm-active) .cm-hmd-frontmatter.cm-def {
  visibility: hidden;
}
.is-live-preview .cm-line .cm-hmd-frontmatter.cm-def .cm-selection {
  visibility: visible;
}

```

**Comment by hua03** [comment]
I had noticed this problem, but didn't have a good idea to deal with it at that time because the plugin will not change the rendering process of markdown. 
I'll try it you provide way.

**Comment by z2k-gwp** [comment]
Awesome - I'll try it out. My YAML frontmatter is some 30 lines long, so it is a miracle to have this..

**Comment by hua03** [comment]
Sorry, it seems has some problems. I'll see how to solve it at the weekend.

**Comment by hua03** [comment]
fixed in v1.2.0
